modules/manage/partials/mountable-topics.adoc

```diff
@@ -1,8 +1,8 @@
-For topics with Tiered Storage enabled, you can mount and unmount topics to transfer the topic data between your cluster and object storage. This allows you to free up and reclaim unused partition space, or migrate a topic to a different cluster and hibernate or decommission the topic or even the entire cluster.
+For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and unmount a topic from object storage to attach the topic data to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up and reclaim system resources taken up by the topic, or migrate a topic to a different cluster.
 
-Redpanda also transfers topic definitions when mounting or unmounting a topic.
+Redpanda also transfers topic manifests when mounting or unmounting a topic, so the topic can quickly accept reads and writes again and you can resume cluster workloads with ease. 
```

Comment:
Wait. If the topic is deleted from the source cluster after unmounting, how does it accept read and writes again? 
Maybe this needs to be expanded.  

Redpanda also transfers topic manifests when mounting or unmounting a topic, making possible to quickly resume operations after re-mounting. (or something to that effect)